app/src/main/java/com/anysoftkeyboard/keyboards/KeyboardSwitcher.java

```diff
@@ -81,7 +81,7 @@
     private static final int SYMBOLS_KEYBOARD_REGULAR_INDEX = 0;
     private static final int SYMBOLS_KEYBOARD_ALT_INDEX = 1;
     private static final int SYMBOLS_KEYBOARD_ALT_NUMBERS_INDEX = 2;
-    private static final int SYMBOLS_KEYBOARD_LAST_CYCLE_INDEX = SYMBOLS_KEYBOARD_ALT_NUMBERS_INDEX;
+    private static final int SYMBOLS_KEYBOARD_LAST_CYCLE_INDEX =  1;
```

Comment:
@VitaliPom what will this change do?

Reply:
@menny it kills Phone Numbers keyboard, I opened an issue for that and closed when I fixed. The keyboard pops up already when necessary and just makes it impossible to use the symbols otherwise. This feature leaves only symbols[0] and symbols[1].